client/my-sites/upgrades/navigation.jsx

```diff
@@ -44,7 +44,7 @@ var NAV_ITEMS = {
 
 	Email: {
```

Comment:
Tho it's a function, so it would change everywhere, but if we have direct links, it won't work.